tests/subworkflows/nf-core/fgbio_create_umi_consensus/main.nf

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { CREATE_UMI_CONSENSUS } from '../../../../subworkflows/nf-core/fgbio_create_umi_consensus/main'
+
+workflow test_fgbio_create_umi_consensus_mem1 {
+    reads          =    [
+                        [ id:'test', single_end:false ], // meta map
+                        [ file(params.test_data['homo_sapiens']['illumina']['test_umi_1_fastq_gz'], checkIfExists: true),
+                        file(params.test_data['homo_sapiens']['illumina']['test_umi_2_fastq_gz'],   checkIfExists: true)
+                        ]
+                        ]
```

Comment:
I find this difficult to read structurally. 
```suggestion
    reads = [
        [ id:'test', single_end:false ], // meta map
        [ 
            file(params.test_data['homo_sapiens']['illumina']['test_umi_1_fastq_gz'], checkIfExists: true),
            file(params.test_data['homo_sapiens']['illumina']['test_umi_2_fastq_gz'], checkIfExists: true)
        ]
    ]
```
I usually format like this, but it's not the cleanest either.

Reply:
no problem, always good to improve readability